### PR TITLE
Update public.sql

### PR DESCRIPTION
**Pull request by eliana-flo**
Added comments for public.circulation_loans table

### CODE REVIEW

File: sql/comments/public.sql [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+COMMENT ON COLUMN public.circulation_loans.renewal_count IS 'Count of how many times a loan has been renewed (incremented by the client)';
+COMMENT ON COLUMN public.circulation_loans.loan_policy_id IS 'ID of last policy used in relation to this loan';
+COMMENT ON COLUMN public.circulation_loans.checkout_service_point_id IS 'ID of the Service Point where the last checkout occured';
+COMMENT ON COLUMN public.circulation_loans.patron_group_id_at_checkout IS 'Patron Group Id at checkout';
+COMMENT ON COLUMN public.circulation_loans.claimed_returned_date IS 'Date and time the item was claimed returned for this loan';
+COMMENT ON COLUMN public.circulation_loans.overdue_fine_policy_id IS 'ID of overdue fines policy at the time the item is check-in or renewed';
+COMMENT ON COLUMN public.circulation_loans.lost_item_policy_id IS 'ID of lost item policy which determines when the item ages to lost and the associated fees or the associated fees if the patron declares the item lost.';
+COMMENT ON COLUMN public.circulation_loans.checkin_service_point_id IS 'Comment to last action performed on a loan';
```
Comment (adoerrer) [inline sql/comments/public.sql:21]: I assume since this is an ID column it can't be a comment.

**Review by adoerrer — CHANGES_REQUESTED** [review]
Except the line 21 all comments looks good. Thank you for helping on that task!

File: sql/comments/public.sql (hunk 1 above)
Comment (amzoss) [reply sql/comments/public.sql:21]: agreed, seems like the comment for checkin_service_point_id should be "ID of the Service Point where the last checkin occurred"

**Review by amzoss — CHANGES_REQUESTED** [review]
looks good except for line 21

**Comment by nassibnassar** [comment]
@eliana-flo I think if you make more commits in this branch, for example to address the suggestions above, this pull request should automatically pick up the new commits and add them here, since they will be in the same branch.

**Comment by eliana-flo** [comment]
> @eliana-flo I think if you make more commits in this branch, for example to address the suggestions above, this pull request should automatically pick up the new commits and add them here, since they will be in the same branch.

Thanks Nassib, I made another commit fixing the line! Sorry for not being a better proof reader :)

**Review by amzoss — APPROVED** [review]
looks great!

**Pull request merged**